updates/3/updates.sql

```diff
@@ -49,4 +49,38 @@ INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingec
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("POLX", "S", "Polylastic", "POLX", 1619730927, NULL, "polylastic", "POLX", "POLX"); INSERT INTO common_asset_details(asset_id, forked) VALUES("POLX", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xF4d861575ecC9493420A3f5a14F85B13f0b50EB3", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xF4d861575ecC9493420A3f5a14F85B13f0b50EB3", "C", "Fractal Protocol Token", "FCL", 1613480889, NULL, "fractal", "FCL", "0xF4d861575ecC9493420A3f5a14F85B13f0b50EB3");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xbDEC45952B5E234EdDC2981B43eeD360826D5087", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xbDEC45952B5E234EdDC2981B43eeD360826D5087", "C", "Mogu Token", "MOGX", 1566834869, NULL, "mogu", "MOGU", "0xbDEC45952B5E234EdDC2981B43eeD360826D5087");
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0x5273063725a43A323300C502478C22FbB4e92C2D", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0x5273063725a43A323300C502478C22FbB4e92C2D", "C", "MY IDENTITY COIN", "MYID", 1614319404, NULL, "my-identity-coin", "MYID", "0x5273063725a43A323300C502478C22FbB4e92C2D");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("NDAU", "W", "Ndau", "NDAU", 1569891600, NULL, "ndau", "XND", "NDAU"); INSERT INTO common_asset_details(asset_id, forked) VALUES("NDAU", NULL);
+*
+INSERT INTO ethereum_tokens(address, decimals, protocol) VALUES("0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", 18, NULL); INSERT INTO assets(identifier, type, name, symbol,started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("_ceth_0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37", "C", "Polka City", "POLC", 1613575886, NULL, "polkacity", "", "0xaA8330FB2B4D5D07ABFE7A72262752a8505C6B37");
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("WIN-4", "P", "WINkLink", "WIN", 1564440570, NULL, "winklink", "", "WIN-4"); INSERT INTO common_asset_details(asset_id, forked) VALUES("WIN-4", NULL);
+*
+INSERT INTO assets(identifier, type, name, symbol, started, swapped_for, coingecko, cryptocompare, details_reference) VALUES("STX", "W", "Stacks", "STX", 1572055200, NULL, "blockstack", "STX", "STX"); INSERT INTO common_asset_details(asset_id, forked) VALUES("STX", NULL);
```

Comment:
This is already in the global DB @yabirgb, with the identifier STX-2. Please before adding an asset make sure to check that we do not have it added already.

Reply:
I only checked by name and not by symbol. I have `STX-2` as Blockstack and not Stacks

Reply:
Check with direct SQL queries. The search in the app is by text and afaik, @kelsos wants to fix this